common/presenters/profile-to-card-component.js

```diff
@@ -6,6 +6,7 @@ const filters = require('../../config/nunjucks/filters')
 const frameworkFlagsToTagList = require('./framework-flags-to-tag-list')
 
 function profileToCardComponent({
+  locationType = undefined,
```

Comment:
You can remove the default value and it will be set to `undefined` too:
```suggestion
  locationType,
```